### DISCUSSION TITLE
Support of Sybase / SQL Anywhere

### DISCUSSION THREAD

**Discussion by KBA2024** [jobrunr/jobrunr#1167] [Q&A] [answered]
Many of our customers use Sybase, and I'm curious why it isn't supported by JobRunr yet. Is this due to the lack of features like 'FOR UPDATE SKIP LOCKED,' or has the opportunity to implement support simply not arisen?

**Comment by rdehuyss** [comment]
Hi @KBA2024 - well, we only support the most used databases in JobRunr OSS as it takes a lot of time to maintain the different SQL scripts for each database.

Would you consider taking a JobRunr Pro subscription if Sybase support would be added to it (and you can try it out for a month or so)?

**Reply by KBA2024** [reply, marked as answer]
I am currently benchmarking various frameworks for job scheduling and will present my findings to the CTO and management, who will make the final decision. Before including JobRunr in the list, I wanted to confirm whether it supports Sybase/SQL Anywhere. I'll note that support is available with the Pro version. It's likely they will ask about pricing, so I will reach out through [your website](https://www.jobrunr.io/) to gather that information.

**Reply by rdehuyss** [reply]
I assume you know Sybase (both ASE and SQL anywhere) is EOL as of next year? I read about conflicting dates, either somewhere in 2025 or in 2028 (FYI - I'm not a Sybase connaisseur)?

**Reply by KBA2024** [reply]
Yes, the End of Mainstream Maintenance for SQL Anywhere is officially scheduled for the end of 2028. We are planning a migration to PostgreSQL, but this transition is not expected to be started before 2026.
